package.json

```diff
@@ -11,13 +11,25 @@
   },
   "devDependencies": {
     "bootstrap-sass": "^3.4.1",
-    "cross-env": "^5.2.0",
-    "jquery": "^3.5.1",
-    "jquery-ui": "^1.12.1",
+    "cross-env": "^7.0.3",
+    "jquery": "^3.6.0",
+    "jquery-ui": "^1.13.1",
     "laravel-mix": "^5.0.9",
-    "resolve-url-loader": "^3.1.2",
-    "sass": "^1.23.7",
+    "resolve-url-loader": "^5.0.0",
+    "sass": "^1.49.8",
     "sass-loader": "^7.3.1",
-    "vue-template-compiler": "^2.6.10"
+    "vue-template-compiler": "^2.6.14"
```

Comment:
Do we use vue.js in this?
see if you can find any references to it.

Reply:
This "Vue.component("example",n(34));new Vue({el:"#app"})}" is in public/store/js/app.js, if that counts.

Reply:
interesting... i'll make a research item to determine if we can pull it out at another time!